lib/ansible/modules/network/cnos/cnos_backup.py

```diff
@@ -33,21 +33,23 @@
 ---
 module: cnos_backup
 author: "Anil Kumar Muraleedharan (@amuraleedhar)"
-short_description: Backup the current running or startup configuration to a remote server on devices running Lenovo CNOS
+short_description: Backup the current running or startup configuration to a
+ remote server on devices running Lenovo CNOS
 description:
     - This module allows you to work with switch configurations. It provides a
      way to back up the running or startup configurations of a switch to a
      remote server. This is achieved by periodically saving a copy of the
      startup or running configuration of the network device to a remote server
-     using FTP, SFTP, TFTP, or SCP. The first step is to create a directory from
+     using FTP, SFTP, TFTP, or SCP. The first step is to create directory from
```

Comment:
Did you mean to drop the word 'a' here?

Reply:
Will add the 'a' back

Reply:
Will also remove the unwanted code from cnos.py along with this PR